R/PMP.R

```diff
@@ -0,0 +1,315 @@
+                (Sn_m_div*(134.14*n_PMP^-0.053)+(+-0.0006781*n_PMP^2 + 0.01089*n_PMP + 0.3812))/100}
+
+        Fig_4.3<-Fig_4.3f(n_PMP,Sn_m_div)
+
+        #Fig 4.4
+        Fig_4.4m<-function(n_PMP){
+                #Expression over 50 years in smaller than 1 - Corrected with max
+                pmax(1,(99.7196936977294+(107.866021487998-99.7196936977294)/
+                                (1+(n_PMP/13.0261452397594)^2.25690664033008))/100)
+        }
+
+        Fig_4.4s<-function(n_PMP){
+                #Expression over 50 years in smaller than 1 - Corrected with max
+                pmax(1,(741.591367770118*exp(-n_PMP/2.63780903040506)+24.8370732678786*
+                                exp(-n_PMP/20.9101626977193)+97.9173078131786)/100)
+
+        }
+
+        Fig_4.4<-c(Fig_4.4m(n_PMP),Fig_4.4s(n_PMP))
+
+        #Fig 4.1
+        Km.f.24<-function(X.mean){
+                0.0000000001638*X.mean^4 - 0.0000002397*X.mean^3 + 0.0001439*X.mean^2 - 0.05951*X.mean + 19.98}
+        Km.f.6<-function(X.mean){
+                -0.000000000038701397*X.mean^5 + 0.000000031987514*X.mean^4 - 0.000010135438*X.mean^3 +
+                        0.0015812728*X.mean^2 - 0.16896843*X.mean + 19.951655}
+        Km.f.1<-function(X.mean){
+                +-0.0000086920029*X.mean^3 + 0.0023070606*X.mean^2 - 0.29365613*X.mean + 19.971686}
+
+        if (input.hrs==24){Km.f<-Km.f.24}
+
+        if (input.hrs==6){Km.f<-Km.f.6}
+
+        if (input.hrs==1){Km.f<-Km.f.1}
+
+        Km<-Km.f(X.mean[[1]])
+
+        X.n_adj<-X.mean[[1]]*Fig_4.2*Fig_4.4[1]
+        S.n_adj<-X.sd[[1]]*Fig_4.3*Fig_4.4[2]
+
+        PMP1<-X.n_adj+Km*S.n_adj
+
+        #Fig 4.5
+        nou.f<-function(number.of.obs.unit){
+                (100.2766207711+(684.0989213113-100.2766207711)/
+                         (1+(number.of.obs.unit/0.03150054213)^1.116899589012))/100}
+
+        PMP2<-PMP1*nou.f(number.of.obs.unit)
+
+        #Fig 4.7
+        area_corr.f.24<-function(area.km){
+                (-1.13762590487202E-13*area.km^5+3.43734418674257E-10*area.km^4+
+                         -4.00454902021794E-07*area.km^3+0.000226801496747288*area.km^2+
+                         -0.0710318092970015*area.km+101.210146730303)/100}
+        area_corr.f.6<-function(area.km){
+                (+-901.2243442608+(113.7869909952--901.2243442608)/
+                         (1+(area.km/1796029203.58)^0.2343007717958))/100}
+
+        area_corr.f.1<-function(area.km){
+                (47.13563121645+(126.0011250857-47.13563121645)/
+                         (1+(area.km/69.77177029402)^0.5654945864788))/100}
+
+        if (input.hrs==24){area_corr<-area_corr.f.24(area.km)}
+
+        if (input.hrs==6){area_corr<-area_corr.f.6(area.km)}
+
+        if (input.hrs==1){area_corr<-area_corr.f.1(area.km)}
+
+        #For small area is corrected to a a maximum of 1
+        area_corr<-pmin(area_corr,1)
+
+        PMP3<-PMP2*area_corr
+
+        ####FIGURES######
+        if(graphs==T){

```

Comment:
It is best to use `TRUE` rather than simply `T` to avoid ambiguity.

Reply:
agreed.